### DISCUSSION TITLE
2-way Office 365 Outlook sync

### DISCUSSION THREAD

**Discussion by dawidurbanski** [wpappointments/wpappointments#50] [Ideas]
It should allow for:

- Adding one or many calendars

For each calendar:
- Should block all time that has events in the connected calendar
    - as option
    - enabled by default
- Should add new events to the calendar
   - as option
   - enabled by default
   - adds to first calendar in the list by default
   - allows to pick to which calendars to add events

**Comment by klartext79** [comment]
If you allow adding more than one calendar e.g. one Gmail and one Outlook (private and business) you should add an option to make one of them the primary calendar where appointments are booked into (like calendly does) @dawidurbanski

**Comment by wppoland** [comment]
Great point about primary calendar selection. When we build multi-calendar support, we will make sure there is a clear way to designate which calendar receives new bookings. Thanks for the feedback!
